tests/suites/cloud/suite.js

```diff
@@ -189,13 +190,17 @@ module.exports = {
       ),
     });
 
-    // unpack OS
-    await this.context.get().os.fetch();
 
-    // If this is a flasher image, and we are using qemu, unwrap
+    this.suite.context.set({
+			workerContract: await this.context.get().worker.getContract()
+		})
+		// Unpack OS image .gz
+		await this.context.get().os.fetch();
+
+		// if we are running qemu, and the device type is a flasher image, we need to unpack it from the flasher image to get it to boot
 		if (
 			this.suite.deviceType.data.storage.internal &&
-			process.env.WORKER_TYPE === `qemu`
+			this.context.get().workerContract.workerType === `qemu`
```

Comment:
`context.get()` is no longer necessary, you can access context properties directly from the Test object now, so this would be simply `this.workerContract.workerType`

Reply:
Ditto for the other occurrences